packages/cfd/README.md

```diff
@@ -1,97 +1,45 @@
-# Deriv App
+# CFD (contract for difference)
 
-This repository contains the static HTML, Javascript, CSS, and images content of the [Deriv](http://app.deriv.com) website.
+This repository is a workspace of [deriv-app](../../README.md) monorepo and contains the static HTML, Javascript, CSS, and images content of the [Deriv](http://app.deriv.com) website.
+[DMT5](https://app.deriv.com/mt5) and [Deriv X](https://app.deriv.com/derivx) platforms are related to this workspace.
 
-**In this document**
+## How to Install the Project
 
--   [Other documents](#other-documents)
--   [Installation](#installation)
--   [How to work with this project](#how-to-work-with-this-project)
-    -   [Deploy to your gh-pages for the first time](#deploy-to-your-gh-pages-for-the-first-time)
-    -   [Deploy to the root of the gh-pages](#deploy-to-the-root-of-the-gh-pages)
-    -   [Clean root and deploy to it](#clean-root-and-deploy-to-it)
-    -   [Deploy to test folder](#deploy-to-test-folder)
-    -   [Preview on your local machine](#preview-on-your-local-machine)
--   [Miscellaneous](#Miscellaneous)
--   [Selenium tests](#selenium-tests)
+You need to follow the instructions [here](../../README.md).
 
-## Other documents
+## How To Work With This Project
 
--   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding, code usage)
+To run and work on this workspace you need to use `npm run serve cfd` command along with `npm run serve core`.
+It will watch `cfd` so after you made a change in this package, it will automatically rebuild `cfd` and recompile `core`.
```

Comment:
```suggestion
Webpack will watch changes in `cfd` so that if you made any changes in this package, it will automatically rebuild `cfd` and recompile `core`.
```